spec/controllers/admin/variant_overrides_controller_spec.rb

```diff
@@ -3,6 +3,29 @@
 require 'spec_helper'
 
 RSpec.describe Admin::VariantOverridesController, type: :controller do
+  describe "index" do
+    context "not logged in" do
+      it "redirects to login" do
+        pending
+        get :index
+        expect(response).to have_http_status :found
```

Comment:
```suggestion
        expect(response).to redirect_to xxx_path
```

Reply:
I skipped this before because it's not a simple path and it didn't seem too important. But seeing as you've mentioned it, I've updated it now 🙂